client/electron/electron-builder.json

```diff
@@ -32,30 +34,32 @@
     "files": [
       "client/electron/linux_proxy_controller/dist",
       "client/electron/icons/png",
-      "client/output/build/linux",
-      "!client/output/build/linux/*.h"
+      "output/client/linux-amd64",
```

Comment:
I would prefer to not annotating the output folder with the arch. Let's imagine we have a new argument in the build script: `npm run action build linux arm` or `npm run action build linux x64`. Gomobile will generate the specified arch binary into `output/client/linux` without the arch in the folder name. In this case, `electron-builder.json` doesn't need to change at all because the binary paths are identical.